### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/cool-certificate-manager-iam.git cool-certificate-manager-iam
    cd cool-certificate-manager-iam
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md examples/basic_usage/README.md examples/basic_usage/main.tf examples/basic_usage/outputs.tf examples/basic_usage/variables.tf main.tf outputs.tf variables.tf 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: providers.tf [hunk 1]
```diff
@@ -3,15 +3,21 @@
 # the other providers.
 
 provider "aws" {
+  default-tags {
```
Comment (dav3r) [inline providers.tf]:
```suggestion
  default_tags {
```

File: providers.tf [hunk 2]
```diff
@@ -3,15 +3,21 @@
...
 # The provider used to create resources inside the Users account.
 provider "aws" {
-  alias  = "users"
-  region = var.aws_region
+  alias = "users"
   assume_role {
     role_arn     = data.terraform_remote_state.users.outputs.provisionaccount_role.arn
     session_name = local.caller_user_name
   }
+  default-tags {
```
Comment (dav3r) [inline providers.tf]:
```suggestion
  default_tags {
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Not today Satan!  😈

**Review by dav3r — APPROVED** [review]
🐸

File: README.md [hunk 3]
```diff
@@ -52,32 +52,54 @@ COOL certificates.
...
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"The IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account."` | no |
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"DNS-AssumeRoute53ResourceChange-cyber.dhs.gov"` | no |
+| aws\_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `"us-east-1"` | no |
+| certificate\_managers\_group\_name | The name of the group to be created for certificate manager users. | `string` | `"certificate_managers"` | no |
 | tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
```
Comment (mcdonnnj) [inline README.md]: The variable description would need to be updated as well.

```suggestion
| tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
```

File: README.md [hunk 4]
```diff
@@ -52,32 +52,54 @@ COOL certificates.
...
+| assume\_dns\_certificatesbucketfullaccess\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows full access to the certificates bucket in the DNS account. | `string` | `"DNS-AssumeCertificatesBucketFullAccess"` | no |
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"The IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account."` | no |
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"DNS-AssumeRoute53ResourceChange-cyber.dhs.gov"` | no |
+| aws\_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `"us-east-1"` | no |
```
Comment (mcdonnnj) [inline README.md]: The variable description would need to be updated as well.

```suggestion
| aws\_region | The AWS region to deploy into (e.g. us-east-1). | `string` | `"us-east-1"` | no |
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ Two period suggestions is all I have.

File: README.md (hunk 4 above)
Comment (jsf9k) [reply README.md]: See commit dc20303.

File: README.md (hunk 3 above)
Comment (jsf9k) [reply README.md]: See commit dc20303.

**Pull request merged**